<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-23T23:11:07 ICT" name="Default suite" finished-at="2024-09-23T23:11:10 ICT" duration-ms="3446">
    <groups>
    </groups>
    <test started-at="2024-09-23T23:11:07 ICT" name="Default test" finished-at="2024-09-23T23:11:10 ICT" duration-ms="3446">
      <class name="testCICD.loginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:07 ICT" name="setUp" finished-at="2024-09-23T23:11:08 ICT" duration-ms="1622" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testInvalidLogin()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:08 ICT" name="testInvalidLogin" description="Test login form with invalid credentials" finished-at="2024-09-23T23:11:09 ICT" duration-ms="327" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Thông báo lỗi không đúng! expected [Expected error message] but found [Invalid email or password]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Thông báo lỗi không đúng! expected [Expected error message] but found [Invalid email or password]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at testCICD.loginTest.testInvalidLogin(loginTest.java:58)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:09 ICT" name="tearDown" finished-at="2024-09-23T23:11:09 ICT" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:09 ICT" name="setUp" finished-at="2024-09-23T23:11:10 ICT" duration-ms="974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testLogin()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:10 ICT" name="testLogin" description="Test login form with valid credentials" finished-at="2024-09-23T23:11:10 ICT" duration-ms="407" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCICD.loginTest@57a3e26a]" started-at="2024-09-23T23:11:10 ICT" name="tearDown" finished-at="2024-09-23T23:11:10 ICT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCICD.loginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
